src/App.scss

```diff
@@ -1,15 +1,8 @@
 .App {
-  display: flex;
   height: 100vh;
 }
 
-.App-header {
-  width: 30vw;
-  background-color: #282c34;
-  color: white;
-}
-
 .App-main {
-  flex: 1;
-  overflow-y: auto;
```

Comment:
Does the main content still scroll? e.g. What happens if you have tons of keys in RTDB Viewer?

Reply:
Yep, so App-main has a height of the viewport minus the app-bar. Each Route-component can set its own scroll pattern... e.g. setting the route-component to height:100%;overflow:scroll; would give you the same effect as today; otherwise the route-component still scrolls but renders the whole node